### DISCUSSION TITLE
Does it support YOLOv8 and oriented bounding box (OBB)

### DISCUSSION THREAD

**Discussion by YuancXie** [PUTvision/qgis-plugin-deepness#145] [Q&A]
YOLOv8 now supports oriented bounding box (OBB). I was wondering if this is supported by Deepness as well. Thank you.

**Comment by bartoszptak** [comment]
Interesting idea, we will think about that, but we do not plan to implement it in the near future. 

Contributions and pull requests are welcome!
[General Information for Developers](https://qgis-plugin-deepness.readthedocs.io/en/latest/dev/dev_general_info.html)

**Reply by bartoszptak** [reply]
Hi @YuancXie ! OBB is now available with YOLO11!
